src/database/table.ts

```diff
@@ -36,4 +38,67 @@ export default class Table {
 
     return content.runSQL(sql);
   }
+
+  static getRelatedObjects(schema: string, name: string): void {
+    const content = `SELECT SQL_OBJECT_TYPE, SCHEMA_NAME, SQL_NAME, LIBRARY_NAME,
+              SYSTEM_NAME, OBJECT_OWNER, LONG_COMMENT, OBJECT_TEXT, LAST_ALTERED 
+              FROM TABLE(SYSTOOLS.RELATED_OBJECTS('${schema}', '${name}')) ORDER BY SQL_NAME`;
```

Comment:
I would suggest the following column order:
```
SQL_NAME, SYSTEM_NAME, SCHEMA_NAME, LIBRARY_NAME, SQL_OBJECT_TYPE, 
              OBJECT_OWNER, LAST_ALTERED, OBJECT_TEXT, LONG_COMMENT
```
Would also order by SQL_OBJECT_TYPE then by SQL_NAME